### PR TITLE
Follow-up edits to PR#2814

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/2814

### CODE REVIEW

**Comment by ahardin-rh** [comment]
Preview build: file:///home/ahardin/openshift-docs/_preview/openshift-enterprise/PR2814-followup/install_config/install/prerequisites.html#system-requirements

**Comment by ahardin-rh** [comment]
Also addresses https://github.com/openshift/openshift-docs/issues/2688

**Comment by ahardin-rh** [comment]
@jeremyeder @timothysc PTAL

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -88,22 +88,25 @@ endif::[]
...
+* NetworkManager 1.0 or later
+* 1 vCPU.
+* Minimum 8 GB RAM.
+* Minimum 15 GB hard disk space for the file system containing *_/var/_*.
+* An additional minimum 15 GB unallocated space to be used for Docker's storage
 back end; see xref:configuring-docker-storage[Configuring Docker Storage]
 below.
+
+|External etcd Nodes
+|Minimum 20 GB hard disk space.
```
Comment (jeremyeder) [inline install_config/install/prerequisites.adoc]: I would say ... "Minimum 20 GB hard disk space for etcd data"...just be a little more explicit.  Fair?

**Review by jeremyeder — CHANGES_REQUESTED** [review]

**Comment by ahardin-rh** [comment]
@jeremyeder Fixed. Thanks! Merging!

**Comment by ahardin-rh** [comment]
Rev history in original PR

**Pull request merged**